src/theory/quantifiers/ceg_t_instantiator.cpp

```diff
@@ -861,8 +1035,35 @@ bool BvInstantiator::processAssertion( CegInstantiator * ci, SolvedForm& sf, Nod
     }
   }
   */
+}
 
+bool BvInstantiator::processEquality( CegInstantiator * ci, SolvedForm& sf, Node pv, std::vector< Node >& term_coeffs, std::vector< Node >& terms, unsigned effort ) {
+  //processLiteral( ci, sf, pv, terms[0].eqNode( terms[1] ), effort );
```

Comment:
Is it somehow possible to get a minimal version of processLiteral that could be commented in or is there some other way that the existing processLiteral could at least be sanity tested?

Reply:
I want this commit to add the infrastructure only and have zero impact on actual performance.  I internally archive StarExec jobs (and CVC4 binaries) for all commits that change the performance of master, which takes some time. But I do not intend for this one to have any effect.